tests/configs/setupHooks.js

```diff
@@ -0,0 +1,13 @@
+async function postSetup() {
+    await mockAsyncFunction();
+}
+
+function mockAsyncFunction() {
+    return new Promise((resolve, reject) => {
+        resolve();
+    });
+}
+
+module.exports = {
+    postSetup
+};
```

Comment:
Maybe export an empty object and mock as required implementation, or leave as it is so it can be used for examples?